examples/simpleapp/mc_sc_workflow_example.md

```diff
@@ -164,12 +163,12 @@ As a result id of block with Sidechain transaction will be printed:
 Now mainchain contains created Sidechain. 
 
 **Step 9: Get information from mainchain for forming genesis Sidechain block**
-Sidechain now is created in mainchain, but for starting Sidechain genesis Sidechain block shall be formed and put to configuration file, as well as some additional parameters.
+Sidechain now is created in mainchain, but for starting Sidechain configuration file must be supplemented with genesis Sidechain block and with some additional parameters.
 Genesis info for further using in ScBootstrappingTool could be extracted by next command:
 ```
 $ ./zen-cli -regtest getscgenesisinfo 2f7ed2e07ad78e52f43aafb85e242497f5a1da3539ecf37832a0a31ed54072c3
 ```
-Where `"2f7...2c3"` is Sidechain id from step 6. As a result we get byte sequence which will be passed to Bootstrap tool.
+Where `"2f7...2c3"` is Sidechain id from step 7. As a result we get byte sequence which will be passed to Bootstrap tool.
 
 Example output:
 ```
```

Comment:
and in step 10: `"secret": "003...2ac"` any key (included one generated on step 3) -> `"secret": "003...2ac"` any ed25519 key (included one generated on step 3)